src/Calendar.jsx

```diff
@@ -349,6 +349,7 @@ let Calendar = React.createClass({
     )
 
     let ToolbarToRender = components.toolbar || Toolbar
+    const now = new Date()
```

Comment:
rather than using new Date() for now, we should pass this as a prop, with default value of new Date(). Makes the component slightly more pure.  It allows easily testing the code for highlighting the current time, and also things like displaying a calendar for a different time, or 2 calendars on the same page with different now value.